cis194/week1/robert-chang/Homework01/package.yaml

```diff
@@ -0,0 +1,48 @@
+name:                Homework01
```

Comment:
Based on what I've discovered, looks like most package names out there in the haskell world is lowercased and separated with `-`s. For example `tasty-hspec`, `hpc-coveralls` etc.

Reply:
seems I have to regenerate the project using stack... 😢  
only change the names won't work for me. probably those yaml file need change as well.
I will leave this. for next week's task, I will do better.

Reply:
![Screen Shot 2019-08-09 at 1 21 40 am](https://user-images.githubusercontent.com/4587603/62715552-14bd3f00-ba44-11e9-9c2f-7a08f078df92.png)
@sinogermany  I just checked, homework-01 seems not acceptable by Stack.
so word-xx,  (xx is digits) won't work.